Make the default behavior to ignore minifyEnabled = false variants (#61)
slackhq/keeper: docs/configuration.md

> Keeper's default behavior with no configuration will enable it for only buildTypes which use
> minifyEnabled = true. This may not be what you want for your actual production builds that you

Reviewer: ```suggestion
Keeper's default behavior with no configuration will enable it for only buildTypes that set
`minifyEnabled` to true. This may not be what you want for your actual production builds that you
```

### Variant filter

> By default, Keeper will run on any app variant that sets minifyEnabled = true.

Reviewer: ```suggestion
By default, Keeper will run on any app variant that sets `minifyEnabled` to true.
```
